services/oauth_helpers.py

```
@@ -0,0 +1,186 @@
+        login_hint: Email hint for Google login screen
+        scopes: OAuth scopes (defaults to https://mail.google.com/)
+
+    Returns:
+        str: Complete Google OAuth authorization URL
+
+    Raises:
+        ValueError: If client_id not configured
+    """
+    client_id = client_id or os.getenv("OAUTH_CLIENT_ID")
+    if not client_id:
+        raise ValueError("OAUTH_CLIENT_ID must be configured")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)